src/theme.ts

```diff
@@ -28,6 +28,9 @@ export default {
     typography: {
       link: ColorLink,
       highlight: ColorHighlight,
+      fontFamily: ['"Roboto"', '"Helvetica"', '"Arial"', `sans-serif`].join(
+        ','
+      ),
```

Comment:
Changing the font family for MUI does not necessarily mean it is changed for the page, only MUI components.  If you want the font family for the page to be this value, you'd also need to change it in `public/style.css`